content/v2.0/write-data/use-telegraf/auto-config/_index.md

```diff
@@ -12,11 +12,14 @@ menu:
 weight: 201
 ---
 
-The InfluxDB user interface (UI) provides a workflow that automatically creates
+The InfluxDB user interface (UI) can automatically create
 Telegraf configuration files based on user-selected Telegraf plugins.
 This article describes how to create a Telegraf configuration in the InfluxDB UI and
 start Telegraf using the generated configuration file.
 
+Only a subset of plugins are configurable using the InfluxDB UI.
+To use plugins other than those listed, you must [manually configure Telegraf](/v2.0/write-data/use-telegraf/manual-config).
```

Comment:
This needs to be called out in a note.

~~~
{{% note %}}
Content here
{{% /note %}}
~~~